csp.config.js

```diff
@@ -1,25 +1,26 @@
-/*!
+/*
  * Content-Security-Policy
  */
 const SELF = "'self'"
-const PRODUCION = '*.core.ac.uk'
+// Note it needs to be like that because https://core.ac.uk
+// is not covered by wildcard
+const PRODUCTION = '*.core.ac.uk core.ac.uk'
 
 const config = {
-  'default-src': [SELF, PRODUCION],
+  'default-src': [SELF, PRODUCTION],
   'script-src': [SELF, '*.google-analytics.com'],
-  // TODO: Remove 'unsafe-inline' when the Next.js' bug is resolved
+  // TODO: Move 'unsafe-inline' to development when the Next.js' bug is resolved
   // See more: https://github.com/vercel/next.js/issues/17445
   'style-src': [SELF, "'unsafe-inline'"],
   // Google Analytics may transport data via image:
   // https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#transport
-  'img-src': [SELF, PRODUCION, 'data:', '*.google-analytics.com'],
-  'connect-src': [SELF, PRODUCION, 'sentry.io', '*.google-analytics.com'],
+  'img-src': [SELF, PRODUCTION, 'data:', '*.google-analytics.com'],
```

Comment:
Did I make a mistake? It's terrible! I did it everywhere because I was copying the file.

I think it worth to move CSP to configs repository. However, we will need to develop a way to extend or override the config, what is not good, it requires much effort.